### ISSUE TITLE
Image from group without text 

### ISSUE THREAD

**Issue by pipepa** [Sidneys1/signal-bot#2]
How to make it so that I can receive information about images sent by another person in the group. I noticed that I can see images if they have some text, but without text they are not defined at all. I really need it with the text

```
async def crabby_callback(signal: SignalBot, context: Context, message: DataMessage) -> bool:
        to = context[1]
        
        #print(context[1])
        #print(message.attachments)
        await signal.send_reaction(message, "🦀")
        await signal.send_message(to, f"",
                                  args=SendMessageArgs(attachment="/home/nazik/Стільниця/Створити теку 2/signalbot/example/output.jpg"))
        return True

async def main():
    connection = 'ipc://'

    signal = await create(Account('+4915510680205'), connection)
    #await signal.send_message("+380689053869", "Test")
    signal.on_message(crabby_callback)

    await signal.run()

if __name__ == '__main__':
    asyncio.run(main())
```

**Comment by Sidneys1** [comment]
It seems the error is in the handling of data messages - the assumption is that we never want to process messages without `envelope.dataMessage.message`. On Signal messages without content text (e.g., just an attachment) this field is `null` and thus we exit processing the message early:

https://github.com/Sidneys1/signal-bot/blob/5c84772d0bfcb5371becd52bbbd5248c4ff9168f/src/signal_bot_framework/_signal_impl.py#L178-L180

I will resolve this in the next release.

**Comment by Sidneys1** [comment]
@pipepa this fix has been included in release [v0.2.3-alpha](https://github.com/Sidneys1/signal-bot/releases/tag/v0.2.2-alpha) ([PyPI](https://pypi.org/project/signal-bot-framework/0.2.2a0/)).
